Woof/Woof/Model/Preferences/Preferences.swift

```diff
@@ -0,0 +1,8 @@
+import Foundation
+
+/// Represents user preferences.
+struct Preferences: Codable {
+    /// The selected role for the user.
+    /// If role is not selected returns `nil`.
+    var selectedRole: Role?
```

Comment:
I'd suggest to avoid optional here. Instead you can add a new `none` case in the enum.

Reply:
Agree, it helps us to avoid additional unwrapping in the future.